src/data/officers.json

```diff
@@ -57,11 +33,11 @@
     },
 
     {
-      "name": "Nathan Nguyen",
-      "image": "/static/images/current-officers/nathan_nguyen.jpg",
-      "major": "General Engineering' 24",
-      "position": "Workshops Officer",
-      "description": "Nathan is a big fan of safe, robust systems software. He's typically experimenting with new baking recipes, reading a good book, or consuming an unholy amount of spicy ramen."
+      "name": "Rebekah Garcia",
+      "image": "/static/images/current-officers/rebekah_garcia.jpg",
+      "major": "Finance",
+      "position": "Events and Marketing Officer",
+      "description": "Rebekah is a senior Finance major. In her free time she enjoys road trips, watching movies, and reading."
     }
```

Comment:
Sorry if I'm missing smth, but are Jay and I going to be added later? :0

Reply:
yes, I was waiting for you both to send in your information